miso-web/src/spring-test/java/uk/ac/bbsrc/tgac/miso/webapp/springtest/GroupRestControllerST.java

```diff
@@ -0,0 +1,158 @@
+package uk.ac.bbsrc.tgac.miso.webapp.springtest;
+
+import org.junit.Test;
+
+import org.springframework.web.servlet.*;
+
+import static org.springframework.test.web.servlet.setup.MockMvcBuilders.*;
+
+import javax.ws.rs.core.MediaType;
+
+import org.checkerframework.checker.units.qual.Temperature;
+import org.junit.Before;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.*;
+import org.springframework.test.web.servlet.ResultActions;
+import com.jayway.jsonpath.JsonPath;
+
+import static org.hamcrest.Matchers.*;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+
+import org.springframework.test.web.servlet.MvcResult;
+import com.eaglegenomics.simlims.core.Group;
+import uk.ac.bbsrc.tgac.miso.dto.GroupDto;
+import uk.ac.bbsrc.tgac.miso.dto.Dtos;
+
+import org.springframework.scheduling.annotation.EnableAsync;
+import org.springframework.web.servlet.View;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import org.springframework.security.test.context.support.WithMockUser;
+import uk.ac.bbsrc.tgac.miso.core.data.type.StatusType;
+import static org.junit.Assert.*;
+import java.util.Collections;
+
+import java.util.List;
+import java.util.Arrays;
+import java.util.ArrayList;
+
+import org.springframework.test.web.servlet.MockMvc;
+import java.util.Date;
+
+
+public class GroupRestControllerST extends AbstractST {
+
+  private static final String CONTROLLER_BASE = "/rest/groups";
+  private static final Class<Group> controllerClass = Group.class;
+
+  @Test
+  @WithMockUser(username = "admin", password = "admin", roles = {"INTERNAL", "ADMIN"})
+  public void testAddMembers() throws Exception {
+
+    // must be admin to add group members
+    List<Long> ids = new ArrayList<Long>();
+    ids.add(1L);
+    ids.add(4L);
+
+    getMockMvc().perform(post(CONTROLLER_BASE + "/2/users").content(makeJson(ids)).contentType(MediaType.APPLICATION_JSON))
+      .andExpect(status().isOk());
+
+    Group expanded = currentSession().get(controllerClass, 2);
+    assertEquals(expanded.getUsers().size(), 3);    
```

Comment:
Please first assert the original size before modification, or get the original size and compare the modified size to it. (Below as well)

also, the `expected` value is the first argument, with `actual` second